src/Groups/gsets.jl

```diff
@@ -41,7 +41,7 @@ The fields are
     action_function::Function
     seeds
 
-    function GSetByElements(G::T, fun::Function, seeds; closed::Bool = false, check::Bool = true) where {T<:Union{GAPGroup, FinGenAbGroup}}
+    function GSetByElements(G::T, fun::Function, seeds; closed::Bool = false, check::Bool = true) where {T<:Union{GAPGroup, FinGenAbGroup, AbstractAlgebra.Group}}
```

Comment:
```suggestion
    function GSetByElements(G::T, fun::Function, seeds; closed::Bool = false, check::Bool = true) where {T<:Union{AbstractAlgebra.Group, FinGenAbGroup}}
```
GAPGroup is a subtype of AA.Group (also at many other places here)